boards/arm/reel_board/reel_board.dts

```diff
@@ -48,6 +48,10 @@
 	};
 };
 
+&uicr {
+	gpio-as-nreset;
+};
```

Comment:
It would be better to move this to [reel_board.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/2597b9699931f493d8ab8840bf4650ba31fdff9c/boards/arm/reel_board/dts/reel_board.dtsi) so that both reel_board and reel_board_v2 use this setting (currently, it is missing for the latter).

Reply:
done